server/src/main/java/org/apache/iotdb/db/sync/pipedata/DeletionPipeData.java

```diff
@@ -35,14 +36,20 @@
 public class DeletionPipeData extends PipeData {
   private static final Logger logger = LoggerFactory.getLogger(DeletionPipeData.class);
 
+  private String storageGroup;
```

Comment:
This attribute is only used in the ext-pipe sender and is an unnecessary field in the data transfer. Adding the field will cause additional network overhead and is not recommended.

Reply:
Thank @Cpaulyz 
This storageGroup is very important field for ext-pipe.
Because it cant let ext-pipe module and customers' plugins  do data's parallel processing.
So I have to add this new field.